homeassistant/components/binary_sensor/zha.py

```diff
@@ -230,6 +216,15 @@ def __init__(self, **kwargs):
             general.OnOff.cluster_id: self.OnOffListener(self),
             general.LevelControl.cluster_id: self.LevelListener(self),
         }
+        out_clusters = kwargs.get('out_clusters')
+        min_rpt = 0
+        for cluster_id in [general.OnOff.cluster_id,
+                           general.LevelControl.cluster_id]:
+            if cluster_id not in out_clusters:
+                continue
+            cluster = out_clusters[cluster_id]
+            self._attributes_to_report[cluster] = {0: (min_rpt, 600, 1)}
```

Comment:
constants for #'s maybe?

Reply:
@dmulcahey Do you think we really need attribute reporting configuration for the remotes, ie attribute reporting for "output" clusters? Going to leave it here for now, but for "output clusters"/remote devices I think we just need `cluster.bind()`

Reply:
Leave it until we can confirm either way. I think we need them but am not 100% positive.